go/logger/standard.go

```diff
@@ -172,7 +161,9 @@ func (log *Standard) prepareString(
 
 func (log *Standard) Debug(fmt string, arg ...interface{}) {
 	log.internal.Debug(fmt, arg...)
-	log.logToExternalLoggers(keybase1.LogLevel_DEBUG, fmt, arg)
+	if log.handler != nil {
+		log.handler.Log(keybase1.LogLevel_DEBUG, fmt, arg)
```

Comment:
Oh I thought at some point they were fixed?  Well we can fix this, I wonder if  we should do so now.